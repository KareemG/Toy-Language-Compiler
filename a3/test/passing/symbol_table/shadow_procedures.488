{
    procedure test
    {
        write "Test"
    }

    function test2 : integer
    {
        procedure test
        {
            write "Test2"
        }

        test

        return with 5
    }

    var x : integer

    x := test2
}