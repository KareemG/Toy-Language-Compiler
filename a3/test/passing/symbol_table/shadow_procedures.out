{
 Symbols:
 x: integer
 test2: integer
 test: null
 test{
 Symbols:
 }
 test2 : integer{
 Symbols:
 test: null
 test{
 Symbols:
 }
 }